Mathlib/Combinatorics/SimpleGraph/Triangle/Removal.lean

```diff
@@ -0,0 +1,178 @@
+/-
+Copyright (c) 2022 Yaël Dillies, Bhavik Mehta. All rights reserved.
```

Comment:
```suggestion
Copyright (c) 2024 Yaël Dillies, Bhavik Mehta. All rights reserved.
```

Reply:
This code really was written in 2022 with minimal modifications since. Shouldn't the copyright be the time at which the code is written, not the one at which it is contributed to mathlib? In practice, both often coincide by at most a few months.

Reply:
I don't know... but I am sure in practice it doesn't matter, let's keep 2022.